src/components/AutoComplete/AutoComplete.component.tsx

```diff
@@ -158,17 +197,20 @@ export const AutoComplete = (props: AutoCompleteProps) => {
 				}}
 				{...restProps}
 			/>
-			{options.length && (
+			{!!options.length && (
```

Comment:
`!!` is doing the same thing as `!!false = !true = false`, why are we doing this?

Reply:
when using `{options.length && ` if the options array was empty it was showing 0 on UI. That's why have to convert `options.length` condition to boolean either by `options.length > 0` or `!!options.length`

Reply:
see if this works `!Boolean(options.length)`